lib/multilingual/translation/file.rb

```diff
@@ -29,7 +29,13 @@ def save(translations)
     return result if result[:error]
 
     file = format(processed[:translations])
-    File.open(path, 'w') { |f| f.write file.to_yaml }
+    unless @type == :server
+      File.open(path, 'w') { |f| f.write file.to_yaml }
+
+      config = Rails.application.config
+
+      config.i18n.load_path += Dir[path]
```

Comment:
What happens to this config addition if the file is deleted?

Reply:
![image](https://user-images.githubusercontent.com/35533304/175868287-ea40d5a7-7af6-426f-837d-fa4c9607cc12.png)

I broke the code here, deleted the file, then got this result.